pkg/cloud/aws/services/userdata/userdata.go

```diff
@@ -24,6 +24,23 @@ import (
 )
 
 const (
+	cloudConfigHeader = `## template: jinja
+#cloud-config
+#
+# Copyright 2018 by the contributors
```

Comment:
I don't think we need to output a license header here

Reply:
I removed it.